packages/di/src/lib/injector.ts

```diff
@@ -1,8 +1,10 @@
-import { InjectableMap } from './injectable-map.js';
 import { LifeCycle } from './lifecycle.js';
 import { InjectionToken, Provider, StaticToken } from './provider.js';
 
-export const INJECTABLE_MAP = new InjectableMap();
+/**
+ * Keeps track of all Injectable services and their Injector
```

Comment:
typo: lower case `Injectable`
also I believe `injectors` should be plural in this commet